srcpkgs/reaver/template

```diff
@@ -1,29 +1,20 @@
 # Template file for 'reaver'
 pkgname=reaver
-version=1.6.5
-revision=2
-build_style=gnu-configure
+version=1.6.6
+revision=1
 build_wrksrc="src"
-#conf_files="/etc/reaver/reaver.db"
+build_style=gnu-configure
 makedepends="libpcap-devel"
 short_desc="Brute force attack tool against Wifi Protected Setup (WPS)"
 maintainer="cipr3s <cipr3s@gmx.com>"
-license="GPL-2"
+license="GPL-2.0-or-later"
 homepage="https://github.com/t6x/reaver-wps-fork-t6x"
 distfiles="https://github.com/t6x/reaver-wps-fork-t6x/releases/download/v${version}/${pkgname}-${version}.tar.xz"
-checksum=342e9d265cf459bd2387205b73a63d1fc7582e268f0e9aec20613f3ec11b6a6b
+checksum=e329a0da0b6dd888916046535ff86a6aa144644561937954e560bb1810ab6702
 
 post_extract() {
 	case "$XBPS_TARGET_MACHINE" in
 	*-musl) find -type f -exec sed -i "{}" -e"s;u_char;unsigned char;g" \;
 		;;
 	esac
```

Comment:
Can you check if this is still necessary? Also this should be an easy upstream patch, if you're up for it. `u_char` is just stupid.

Reply:
u_char is stupid but it looks like upstream fixed it a while ago (though not by changing the signature of things but instead making sure to pull in sys/types.h).